daprdocs/content/en/operations/security/mtls.md

```diff
@@ -163,9 +164,52 @@ helm install \
 If the Root or Issuer certs are about to expire, you can update them and restart the required system services.
 
 {{% alert title="Avoiding downtime when rotating certificates" color="warning" %}}
-To avoid downtime when rotating expiring certificates your new certificates must be signed with the same private root key as the previous certificates.
+To avoid downtime when rotating expiring certificates your new certificates must be signed with the same private root key as the previous certificates. This is not currently possible using self-signed certificates generated by Dapr.
+{{% /alert %}}
+
+#### Dapr-generated self-signed certificates
+
+1. Clear the existing Dapr Trust Bundle secret by saving the following YAML to a file (e.g. `clear-trust-bundle.yaml`) and applying this configuration.
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dapr-trust-bundle
+  labels:
+    app: dapr-sentry
+data:
+```
+
+```bash
+kubectl apply -f `clear-trust-bundle.yaml` -n <DAPR_NAMESPACE>
+```
+
+2. Restart the Dapr Sentry service. This will generate a new certificate bundle and update the `dapr-trust-bundle` Kubernetes secret.
+
+```bash
+kubectl rollout restart -n <DAPR_NAMESPACE> deployment/dapr-sentry
+```
+
+3. Once the Sentry service has been restarted, restart the rest of the Dapr control plane to pick up the new Dapr Trust Bundle.
+
+```bash
+kubectl rollout restart deploy/dapr-sentry -n <DAPR_NAMESPACE>
+kubectl rollout restart deploy/dapr-operator -n <DAPR_NAMESPACE>
+kubectl rollout restart statefulsets/dapr-placement-server -n <DAPR_NAMESPACE>
+```
+
+4. Restart your Dapr applications to pick up the latest trust bundle.
+
+{{% alert title="Potential application downtime with mTLS enabled." color="warning" %}}
+Because the newly generated trust bundle uses a different root password mTLS communication with both the control plane and other applications will fail until the applications have been restarted to reload the new Dapr Trust Bundle.
```

Comment:
I think we can be more descriptive here and say that existing running apps will not experience downtime, but at the time of rolling updates and in the cases of service to service invocation, calling apps might fail several times until the certificate on the callee side has been updated.

Reply:
Is there any communication between sidecar and control plane that could be failing? I did not have a chance to observe this.